lib/models/patch/file-patch.js

```diff
@@ -200,6 +274,34 @@ export default class FilePatch {
     }
   }
 
+  /*
+   * Construct a String containing diagnostic information about the internal state of this FilePatch.
+   */
+  inspect(opts = {}) {
```

Comment:
nice!  This is going to be so useful when we are trying to debug the marker layer on a text buffer.